java/src/jmri/jmrit/operations/trains/JmritOperationsTrainsBundle.properties

```diff
@@ -700,8 +700,8 @@ buildCarsNotRoutable = WARNING! The program wasn't able to route the following c
 buildCarNotRoutable = Car ({0}) at ({1}, {2}) final destination ({3}, {4})
 buildNoPickupsFromStaging = No pick ups from staging ({0})
 buildDetermineHpNeeded = Determine if assigned loco ({0}) has the appropriate HP ({1}) based on {2} HP per ton and the route grades
-buildAssignedHpNotEnough = Assigned loco ({0}) HP ({1}) doesn''t meet train requirements ({2}) HP, removing loco from train
-buildAssignedHpTooMuch = Assigned loco ({0}) HP ({1}) exceeds train requirements ({2}) HP, removing loco from train
+buildAssignedHpNotEnough = Assigned loco ({0}) with ({1}) HP doesn''t meet train requirements {2} HP, removing loco from train
```

Comment:
Did you mean to remove the parentheses from {2} here but not in bundles for other locales?